librad/Cargo.toml

```diff
@@ -111,6 +111,12 @@ features = ["logging", "dangerous_configuration"]
 version = "1.0"
 features = ["derive"]
 
+# Note: this is not a direct dependency. governor depends on
+# parking_lot, which depends on smallvec. There is vulnerability
+# in v1.0 and so we're forcing cargo to use this version instead.
+[dependencies.smallvec]
+version = "1.6.1"
```

Comment:
Can we say `>= 1.6.1, 1.6`?

Also, how do we know this is no longer needed? Can we know? What can we know? Are we?

Reply:
> Can we say >= 1.6.1, 1.6?

Ya, `>= 1.6.1` would include the fix :+1: 

> Also, how do we know this is no longer needed? Can we know? What can we know? Are we?

lol, I'll include this link https://github.com/antifuchs/governor/issues/60. Once that's in and bumped we could drop this.